chef/automate
# Add info accordions to client and compliance run detail views

This PR updates the metadata info displayed on client run and compliance scan detail views.

**Before**
![Screen Shot 2019-11-25 at 11 44 27 AM](https://user-images.githubusercontent.com/479121/69568525-fe96a880-0f78-11ea-9ecf-154f1742a41f.png)

**After**
![Screen Shot 2019-11-25 at 11 29 00 AM](https://user-images.githubusercontent.com/479121/69567989-ed996780-0f77-11ea-836e-79c4a3e2ed9f.png)
![Screen Shot 2019-11-25 at 11 28 48 AM](https://user-images.githubusercontent.com/479121/69567995-effbc180-0f77-11ea-9257-b367fa2f27f2.png)

**Before**
![Screen Shot 2019-11-25 at 11 43 55 AM](https://user-images.githubusercontent.com/479121/69568521-fccce500-0f78-11ea-9adb-d79fece6a2b2.png)

**After**
![Screen Shot 2019-11-25 at 11 26 39 AM](https://user-images.githubusercontent.com/479121/69568010-f853fc80-0f77-11ea-8f25-b4b111117fed.png)
![Screen Shot 2019-11-25 at 11 26 28 AM](https://user-images.githubusercontent.com/479121/69568017-fa1dc000-0f77-11ea-940e-b9546e0c981b.png)

### :chains: Related Resources

fixes #1406
fixes #1922

### :athletic_shoe: How to Build and Test the Change

- Nav to `/compliance/reports/nodes` and select an item from list to view detail
- See updated detail view w/ info accordion
- Nav to `/infrastructure/client-runs` and select an from list to view detail
- See updated detail view w/ info accordion

### :white_check_mark: Checklist
